app/__init__.py

```diff
@@ -18,6 +19,9 @@ def create_app(config_name):
     application = Flask(__name__)
     application.wsgi_app = ProxyFix(application.wsgi_app)
     application.config.from_object(config[config_name])
+    migrate = Migrate(application, db)
+
+    manager = Manager(db, MigrateCommand)
```

Comment:
Do we want to do this on all apps or just when testing? If we only need it during testing could we add this in the setup?